documentation/asciidoc/topics/adding_backup_locations.adoc

```diff
@@ -0,0 +1,67 @@
+[id='adding_backup_locations-{context}']
+= Adding Backup Locations to Caches
+When you configure {brandname} clusters to perform cross-site replication, you
+can add backup locations to your cache configurations.
+
+.Procedure
+
+. Create cache configurations with identical names for each site.
++
+Cache configurations at each site can use different cache modes and backup
+strategies. {brandname} replicates data based on cache names.
++
+. Configure sites to go offline automatically with the `take-offline` element.
+.. Set a value of `0` or `-1` for the `after-failures` attribute.
+.. Configure a minimum wait time of 10 seconds (10000 milliseconds).
++
+This configuration ensures that nodes do not attempt to replicate data to
+backup locations that are unavailable, which happens when you take sites 
+offline manually or when failures occur.
```

Comment:
when a user invokes the take-offline function, the cluster stops sending requests to the remote site. 
It is only valid when a site crashes.